dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -16,6 +16,28 @@
 
 package io.dockstore.client.cli.nested;
 
+import java.io.File;
```

Comment:
It could be an IDE upgrade knocked it out on my side. But yes, the checkstyle should have caught it too, so a couple of issues